process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -289,15 +289,15 @@
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Keep a copy of the persistent domain configuration even if this host is not the Domain Controller")
+    @Message(id = Message.NONE, value = "Keep a complete copy of the persistent domain configuration even if this host is not the Domain Controller. This parameter will override any ignore-unused-configuration setting in host.xml and attempt to store the complete domain configuration.")
```

Comment:
When we do the HA DC thing, then promoting manually a non-DCC to master will be an edge case and the main use for --backup would be for a slave.

If you'd like to warn though, that's ok. We could always remove that when we get the HA DC thing going.